core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -51,6 +53,8 @@
 
     private static final Set<Integer> DEFAULT_KNOX_PORTS = Set.of(443);
 
+    private static final Set<String> SUPPORTED_CLOUD_PLATFORMS = Set.of(AWS);
```

Comment:
It could be a configurable value to be more future proof.

Reply:
Added `cb.loadBalancer.supportedPlatforms` setting to application.yml.